src/eventHandlers/continuousIntegrationEnd/index.spec.ts

```diff
@@ -310,6 +310,82 @@ describe('continuous integration end event handler', (): void => {
     );
   });
 
+  it('retries fetching pull request information for which mergeable status is unknown', async (): Promise<void> => {
+    expect.assertions(1);
+
+    const firstResponse: Response = {
```

Comment:
Maybe not the scope of this PR, but having a generator for the response would make some of this code shorter.

Reply:
Let's do it separately, it should be updated for all tests like this one

Reply:
Created a task for it